frontends/precinct-scanner/src/screens/setup_scanner_internal_wiring_screen.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {
+  ScreenMainCenterChild,
+  CenteredLargeProse,
+} from '../components/layout';
+
+export function SetupScannerInternalWiringScreen(): JSX.Element {
+  return (
+    <ScreenMainCenterChild infoBar={false}>
+      <CenteredLargeProse>
+        <h1>Scanner Not Detected</h1>
+        <p>
+          There is an internal connection problem, please report to election
+          clerk.
+        </p>
+      </CenteredLargeProse>
+    </ScreenMainCenterChild>
+  );
+}
+
+/* istanbul ignore next */
+export function DefaultPreview(): JSX.Element {
+  return <SetupScannerInternalWiringScreen />;
+}
```

Comment:
Do we use this `DefaultPreview` anywhere? Seems like we might be able to remove it

Reply:
It needs to be added to `PreviewApp`:

```diff
diff --git a/frontends/precinct-scanner/src/preview_app.tsx b/frontends/precinct-scanner/src/preview_app.tsx
index 4a33cfa6b..ae2456535 100644
--- a/frontends/precinct-scanner/src/preview_app.tsx
+++ b/frontends/precinct-scanner/src/preview_app.tsx
@@ -21,6 +21,7 @@ import * as ScanProcessingScreen from './screens/scan_processing_screen';
 import * as ScanSuccessScreen from './screens/scan_success_screen';
 import * as ScanWarningScreen from './screens/scan_warning_screen';
 import * as SetupPowerPage from './screens/setup_power_page';
+import * as SetupScannerInternalWiringScreen from './screens/setup_scanner_internal_wiring_screen';
 import * as SetupScannerScreen from './screens/setup_scanner_screen';
 import * as UnconfiguredElectionScreen from './screens/unconfigured_election_screen';
 
@@ -45,6 +46,7 @@ export function PreviewApp(): JSX.Element {
         ScanSuccessScreen,
         ScanWarningScreen,
         SetupPowerPage,
+        SetupScannerInternalWiringScreen,
         SetupScannerScreen,
         UnconfiguredElectionScreen,
       ]}
```

Reply:
Then it will show up like this at http://localhost:3000/preview:

<img width="462" alt="image" src="https://user-images.githubusercontent.com/1938/178334259-ff50faa4-9957-4643-bd84-387de01806ce.png">